setup/setup.sh

```diff
@@ -1,8 +1,19 @@
 #!/bin/bash -e
 
-cp setup/package.json .
-cp setup/composer.json .
-cp setup/gulpfile.js .
-cp setup/htaccess-local .htaccess
-cp setup/deploy.sh .
-cp setup/deploy-exclude-list.txt .
+customized_files=(
+  'package.json'
+  'composer.json'
+  'gulpfile.js'
+  'deploy.sh'
+  'deploy-exclude-list.txt'
```

Comment:
personally, I would alphabetize these. Its not a long list, but if it grows, it is easier to manage when it grows if it is alphabetical.